packages/d3fc-axis/src/axisOrdinal.js

```diff
@@ -37,10 +40,117 @@ const axisOrdinal = (orient, scale) => {
         return { offset: [0, base.tickPadding()] };
     };
 
+    const decorateRotation = s => {
+        if (labelRotate) {
+            const { rotate, labelHeight, anchor } = rotation;
+
+            const text = s.select('text');
+            const existingTransform = text.attr('transform');
+
+            const offset = sign() * Math.floor(labelHeight / 2);
+            const offsetTransform = isVertical() ? `translate(${offset}, 0)` : `translate(0, ${offset})`;
+
+            text.style('text-anchor', anchor)
+                .attr('transform', `${existingTransform} ${offsetTransform} rotate(${rotate} 0 0)`);
+        }
+    };
+
+    const measureLabels = s => {
+        const labels = scale.domain();

```

Comment:
This is an ordinal-specific way of obtaining the labels. Is there a generic approach that works for all scale types?